docs/7.x/selinux.md

```diff
@@ -0,0 +1,265 @@
+Current mode:                   enforcing
+Policy from config file:        targeted
+...
+```
+
+Next, the Linux user performing the installation needs to be mapped to the proper SELinux user/role.
+Installer needs to run with an administrative role capable of loading SELinux policy modules - for example `sysadm_r`.
+
+To check existing mappings, use the following:
+
+```sh
+$ semanage login -l
+                Labeling   MLS/       MLS/
+SELinux User    Prefix     MCS Level  MCS Range                      SELinux Roles
+root            user       s0         s0-s0:c0.c1023                 staff_r sysadm_r system_r unconfined_r
+staff_u         user       s0         s0-s0:c0.c1023                 staff_r sysadm_r system_r unconfined_r
+sysadm_u        user       s0         s0-s0:c0.c1023                 sysadm_r
+unconfined_u    user       s0         s0-s0:c0.c1023                 system_r unconfined_r
+...
+```
+
+In order to map the Linux user `john` to the SELinux user `staff_u`, do the following:
+```sh
+$ semanage login -a -s staff_u john
+```
+or modify the mapping with:
+```
+$ semanage login -m -s staff_u john
+```
+
+Switch to the `sysadm_r` role:
+```sh
+$ sudo --role sysadm_r --login
+```
+
+Alternatively, directly run the installer using the role `sysadm_r` and type `sysadm_t`:
+```sh
+$ runcon --role sysadm_r --type sysadm_t ./gravity install --selinux ...
+```
+
+Gravity supports SELinux users `sysadm_u` and `unconfined_u` (and their corresponding roles) out of the box.
+If you need to install and manage Gravity clusters using a different user/role, a custom user policy is required.
+
+
+## Installation
+
+Installer will use SELinux if a) the host has SELinux enabled and b) installer has SELinux support for the host OS distribution and
+c) SELinux support has been turned on with `--selinux`.
+
+Installer does the following as the first step when running on a host with the above conditions met:
+
+  * loads the Gravity SELinux policy module
+  * creates local port bindings for Gravity and Kubernetes-specific ports
+  * creates local file contexts for paths used during the operation
+  * creates local file contexts for custom state directory (if it has been overridden with `--state-dir`)
+
+Additional SELinux configuration might happen later as part of execution of the operation plan.
+
+To start the installation, use the `gravity install` command as usual:
+
+```sh
+$ gravity install --selinux ...
+ Bootstrapping installer for SELinux
+ ...
+```
+
+Likewise, on the joining node:
+
+```sh
+$ gravity join --selinux ...
+ Bootstrapping installer for SELinux
+ ...
+```
+
+SELinux support is managed per-node.
+
+## Upgrades
+
+The upgrade will automatically determine whether SELinux support is on on the cluster nodes and will install and configure
+the new policy individually on each node.
+
+
+## Kubernetes
+
+Kubernetes comes with SELinux support in the form of `seLinuxOptions` inside the `securityContext`.
```

Comment:
Is this a specific K8s version, or just the latest?

Reply:
The idea was to reference the version used in the latest Gravity version (`1.17.x` here) - I'll provide a visible reference to documentation.